ambari-server/pom.xml

```diff
@@ -1776,7 +1776,7 @@
     <dependency>
       <groupId>org.apache.ambari</groupId>
       <artifactId>ambari-serviceadvisor</artifactId>
-      <version>1.0.0.0-SNAPSHOT</version>
+      <version>${project.version}</version>
```

Comment:
this literally breaking everything, service advisor project is not connected to root project and do not aware about ${project.version}